BlackNectar/BlackNectar/StoresTableViewController.swift

```diff
@@ -190,3 +192,44 @@ class StoresTableViewController: UITableViewController, SideMenuFilterDelegate {
     }
     
 }
+
+extension StoresTableViewController {
```

Comment:
By the way, if nothing in this extension will be used outside this file, then you can mark the extension as `fileprivate`. Then all of the methods defined inside of it are not visible outside this file.